pkg/network/server.go

```diff
@@ -266,9 +268,14 @@ func (s *Server) ID() uint32 {
 	return s.id
 }
 
-// Start will start the server and its underlying transport. Calling it twice
-// is an error.
+// Start will start the server and its underlying transport. Can be called
+// only once. The same instance of the Server can't be started again after
+// it's been stopped.
```

Comment:
Just replace `error` with `no-op`: `Calling it twice is a no-op.` It's kind of common phrase, at least for NeoGo.

Reply:
https://en.wikipedia.org/wiki/NOP_(code)